mcp_host_app.py

```
@@ -0,0 +1,256 @@
+                                f"📖 Title: {r['result']['title']}\n"
+                                f"📄 PDF: `{r['result']['pdf_path']}`\n"
+                                f"🧩 Type: {r['result']['puzzle_type']} ({r['result']['count']} puzzles)\n"
+                                f"🔗 Download URL: {r['result']['download_url']}\n"
+                                f"📧 Perfect for email capture!"
+                            )
+                        elif r["tool"] == "get_book_stats":
+                            books = r["result"]["books"]
+                            response_parts.append(
+                                f"📊 **Book Statistics**\n"
+                                f"📚 Total Books: {r['result']['total_books']}\n"
+                                + (f"\n📖 Recent Books:\n" + "\n".join([
```

Flake8 (7.2.0)

[error] f-string is missing placeholders

(F541)